.env.sample

```diff
@@ -1,6 +1,2 @@
 ALTOOL_API_KEY=TBA
 ALTOOL_API_ISSUER=TBA
-# test vm electrs endpoint
-ELECTRS_ENDPOINT=35.189.57.114:50000
-# test vm coordinator
-COORDINATOR_P2P_ENDPOINT=026e1de0e76902d71d2c27bfdda9d48c19c4c550f08bb4189d7208246e0e3af97e@35.189.57.114:9045
```

Comment:
why are we removing these? are these not useful anymore? I suspect that you might need to change some documentation to reflect that, because if I remember correctly they were described earlier

Reply:
They are not useful - if they change they change globally. 
If this is in your local `.env` file everybody has to change the values if the values change. (the file that is checked in is just `.env.sample` - it's just a sample...)

The `.env` file is meant to be for things that are specific to each machine (e.g. signing keys), not the global settings.

Reply:
I was referring to the fact that they were in the sample and were meant to be used / overriden. as such, adding capability to release on Android has nothing to do with removing these?